handlers/dashboard.go

```diff
@@ -344,49 +349,89 @@ func DashboardDataProposals(w http.ResponseWriter, r *http.Request) {
+		return
+	}
+	start, err := strconv.ParseUint(q.Get("start"), 10, 64)
+	if err != nil {
+		logger.Errorf("error converting datatables start parameter from string to int: %v", err)
+		http.Error(w, "Internal server error", http.StatusInternalServerError)
+		return
+	}
 
-	err = db.ReaderDb.Select(&proposals, `
-		SELECT slot, status
-		FROM blocks
-		WHERE proposer = ANY($1)
-		ORDER BY slot`, filter)
+	orderColumn := q.Get("order[0][column]")
+	orderByMap := map[string]string{
+		"0": "validatorindex",
+		"1": "block_slot",
+		"2": "block_slot",
+		"3": "withdrawalindex",
+		"4": "address",
+		"5": "amount",
+	}
+	orderBy, exists := orderByMap[orderColumn]
+	if !exists {
+		orderBy = "validatorindex"
+	}
+	orderDir := q.Get("order[0][dir]")
+	if orderDir != "asc" {
+		orderDir = "desc"
+	}
+
+	length := uint64(10)
+
+	withdrawalCount, err := db.GetDashboardWithdrawalsCount(validators)
 	if err != nil {
-		logger.WithError(err).WithField("route", r.URL.String()).Error("error retrieving block-proposals")
-		http.Error(w, "Internal server error", http.StatusServiceUnavailable)
+		logger.Errorf("error retrieving dashboard validator withdrawals count: %v", err)
+		http.Error(w, "Internal server error", http.StatusInternalServerError)
 		return
 	}
 
-	proposalsResult := make([][]uint64, len(proposals))
-	for i, b := range proposals {
-		proposalsResult[i] = []uint64{
-			uint64(utils.SlotToTime(b.Slot).Unix()),
-			b.Status,
-		}
+	withdrawals, err := db.GetDashboardWithdrawals(validators, length, start, orderBy, orderDir)
+	if err != nil {
+		logger.Errorf("error retrieving validator withdrawals: %v", err)
+		http.Error(w, "Internal server error", http.StatusInternalServerError)
+		return
 	}
 
-	err = json.NewEncoder(w).Encode(proposalsResult)
+	tableData := make([][]interface{}, 0, len(withdrawals))
+
+	for _, w := range withdrawals {
+		tableData = append(tableData, []interface{}{
+			template.HTML(fmt.Sprintf("%v", utils.FormatValidator(w.ValidatorIndex))),
+			template.HTML(fmt.Sprintf("%v", utils.FormatEpoch(utils.EpochOfSlot(w.Slot)))),
+			template.HTML(fmt.Sprintf("%v", utils.FormatBlockSlot(w.Slot))),
+			template.HTML(fmt.Sprintf("%v", utils.FormatTimeFromNow(utils.SlotToTime(w.Slot)))),
+			template.HTML(fmt.Sprintf("%v", utils.FormatAddress(w.Address, nil, "", false, false, true))),
+			template.HTML(fmt.Sprintf("%v", utils.FormatAmount(new(big.Int).Mul(new(big.Int).SetUint64(w.Amount), big.NewInt(1e9)), "ETH", 6))),
+		})
+	}
+
+	data := &types.DataTableResponse{
+		Draw:            draw,
+		RecordsTotal:    withdrawalCount,
+		RecordsFiltered: withdrawalCount,
+		Data:            tableData,
+	}
+
+	err = json.NewEncoder(w).Encode(data)
 	if err != nil {
-		logger.WithError(err).WithField("route", r.URL.String()).Error("error enconding json response")
-		http.Error(w, "Internal server error", http.StatusServiceUnavailable)
+		logger.Errorf("error enconding json response for %v route: %v", r.URL.String(), err)
```

Comment:
It's better practice to use fields, than having everything in one string. But you after the merge of BIDS-1297, I would suggest to always use utils.LogError / utils.LogFatal, which will do all that for you.